docs/UserGuide.md

```diff
@@ -81,12 +83,43 @@ Notes:
 
 * The index must be a positive integer 1, 2, 3, …​
 
+
+
+## Features
+
+### Priority Feature
+
+There are four priority settings to the priority feature - Undefined, Low, Medium and High.
+I4I would show different colored bars at the right of each client depending on the priority assigned to each client.
+
+They are: 
+ 
+|Priority Type   |Syntax  |Picture   |
+|---|---|---|
+|Undefined   |`l/undefined`,`l/u`, `l/U`   |![help message](images/Priority Bar/UndefinedPriority.png)  |
+|Low   |`l/low`,`l/l`, `l/L`    |![help message](images/Priority Bar/LowPriority.png)   |
+|Medium   |`l/medium`,`l/m`, `l/M`    |![help message](images/Priority Bar/MediumPriority.png)   |
+|High   |`l/high`,`l/h`, `l/H`    |![help message](images/Priority Bar/HighPriority.png)   |
+
+Note: 
+* If a client is not assigned a priority, I4I would automatically assign an undefined priority to the client.
+
+* Currently, one can only set the priority on adding the client. 
+
+Example: 
+
+* `add n/Jojo l/h` would add a user named Jojo with a high priority. 
+* `add n/Giorno` would add a user named Giorno with an undefined priority.
+
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ [coming soon]
 
 --------------------------------------------------------------------------------------------------------------------
 
+
+
```

Comment:
Perhaps we can update the command summary table?

Reply:
But this is not a command, thats why i didnt update it.

Reply:
Oh I mean to update the add command with [l/PRIORITY]

![image](https://user-images.githubusercontent.com/31967420/97331535-73264580-18b4-11eb-9c20-88df6a699d88.png)